src/status_im/contexts/shell/qr_reader/view.cljs

```diff
@@ -68,6 +69,34 @@
    [:toasts/upsert invalid-qr-toast]
    300))
 
+(defn- handle-wallet-connect
+  [scanned-text]
+  (let [parsed-uri         (wallet-connect/parse-uri scanned-text)
+        version            (-> parsed-uri :version)
+        expired?           (wc-utils/timestamp-expired? (-> parsed-uri :expiryTimestamp))
```

Comment:
Related to code style, IMO, having a `->` of only one step is not a good idea, since `(f value)` is clear enough and more simple than  `(-> value f)` .

In `expired?` the thread first could also contain `timestamp-expired?`

Reply:
I missed the thread first for expired, fixed it now. thanks <3